### PR TITLE
Cockpit 285

### PR DESCRIPTION
**Pull request by garrett**
Preview: https://garrett.github.io/cockpit-project.github.io/blog/cockpit-285.html

### CODE REVIEW

**Comment by jelly** [comment]
`Support for custom time servers with chronyd` is shown as `merged` while it's actually `closed`.

**Comment by jelly** [comment]
We should probably ignore those `release-notes`.

**Comment by jelly** [comment]
I dropped all old `release-notes` labels.

**Comment by jelly** [comment]
https://github.com/cockpit-project/cockpit/pull/18271
https://github.com/cockpit-project/cockpit/pull/18219

Are two features we miss in the current release notes

File: _posts/2023-02-08-cockpit-285.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+![Screenshot from 2023-02-08 17-59-46]({{ site.baseurl }}/images/285-accounts-group-improvements.png)
+
+Filtering has been added to groups.
+
+![screenshot of group improvements]({{ site.baseurl }}/images/285-accounts-group-improvements-2.png)
+
+## Support a subset of cryptographic policies with subpolicies
+
+Cockpit now supports setting a limited set of cryptographic policies with subpolicies depending on the RHEL/Fedora version. Subpolicies extend a normal cryptographic policy by for example allowing SHA1 or setting stricter requirements such as OSPP.
```
Comment (garrett) [inline _posts/2023-02-08-cockpit-285.md:29]:
```suggestion
A limited set of cryptographic policies with subpolicies are now in the list of cryptographic choices on Red Hat Enterprise Linux, CentOS Stream, Fedora, and related distributions. Subpolicies extend a standard cryptographic policy by either being more permissive or restrictive than the original policy. Examples include allowing SHA1 or using stricter requirements such as OSPP (Operating System Protection Profile, Common Criteria).
```

File: _posts/2023-02-08-cockpit-285.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
+---
+title: Cockpit 285
+author: garrett
+date: '2023-02-08'
+tags: cockpit
+slug: cockpit-285
+category: release
+summary: ''
```
Comment (garrett) [inline _posts/2023-02-08-cockpit-285.md:8]:
```suggestion
summary: Group filtering and creation, cryptographic subpolicies 
```

**Comment by garrett** [comment]
Ugh, capitalization is wrong in the subpolicies...



It really should be FIPS:OSPP, as both are acronyms. I guess that should get a follow-up.

And Legacy:AD-support. And "LEGACY" shouldn't be capitalized.

**Comment by jelly** [comment]
> 



> Ugh, capitalization is wrong in the subpolicies...
> 
> 
> 
> It really should be FIPS:OSPP, as both are acronyms. I guess that should get a follow-up.
> 
> And Legacy:AD-support. And "LEGACY" shouldn't be capitalized.

Let's do that in a follow up :)

File: _posts/2023-02-08-cockpit-285.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 285:
+
+
+## Accounts: Group improvements
```
[the remark this answers is not in the document]
Comment (jelly) [reply _posts/2023-02-08-cockpit-285.md:17]: Right, I guess this title was stolen from the general improvments PR.

File: _posts/2023-02-08-cockpit-285.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+## Try it out
+
+Cockpit 285 is available now:
+
+* [For your Linux system](https://cockpit-project.org/running.html)
+* [Cockpit Client](https://flathub.org/apps/details/org.cockpit_project.CockpitClient)
+
+* [Cockpit Source Tarball](https://github.com/cockpit-project/cockpit/releases/tag/285)
+* [Cockpit Fedora 37](https://bodhi.fedoraproject.org/updates/?releases=F37&packages=cockpit)
+* [Cockpit Fedora 36](https://bodhi.fedoraproject.org/updates/?releases=F36&packages=cockpit)
```
Comment (garrett) [inline _posts/2023-02-08-cockpit-285.md:43]: We need to swap out the links.

Comment (garrett) [inline _posts/2023-02-08-cockpit-285.md:43]:
```suggestion
* [Cockpit Fedora 37](https://bodhi.fedoraproject.org/updates/FEDORA-2023-b2e6448cf5)
* [Cockpit Fedora 36](https://bodhi.fedoraproject.org/updates/FEDORA-2023-4b710b3cf8)
```

File: _posts/2023-02-08-cockpit-285.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+## Try it out
+
+Cockpit 285 is available now:
+
+* [For your Linux system](https://cockpit-project.org/running.html)
+* [Cockpit Client](https://flathub.org/apps/details/org.cockpit_project.CockpitClient)
+
+* [Cockpit Source Tarball](https://github.com/cockpit-project/cockpit/releases/tag/285)
+* [Cockpit Fedora 37](https://bodhi.fedoraproject.org/updates/FEDORA-2023-b2e6448cf5)
+* [Cockpit Fedora 36](https://bodhi.fedoraproject.org/updates/FEDORA-2023-4b710b3cf8)
```
Comment (jelly) [inline _posts/2023-02-08-cockpit-285.md:43]:
```suggestion
* [cockpit-machines Source Tarball](https://github.com/cockpit-project/cockpit-machines/releases/tag/283)
* [cockpit-machines Fedora 37](https://bodhi.fedoraproject.org/updates/FEDORA-2023-e1c4b6e2ef)
* [cockpit-machines Fedora 36](https://bodhi.fedoraproject.org/updates/FEDORA-2023-8f3e0c8d5a)
* [cockpit-podman Source Tarball](https://github.com/cockpit-project/cockpit-machines/releases/tag/62)
* [cockpit-podman Fedora 37](https://bodhi.fedoraproject.org/updates/FEDORA-2023-94d443ecb8)
* [cockpit-podman Fedora 36](https://bodhi.fedoraproject.org/updates/FEDORA-2023-1c6ec9ef3d)
* [cockpit-ostree Source Tarball](https://github.com/cockpit-project/cockpit-ostree/releases/tag/192)
* [cockpit-ostree Fedora 37](https://bodhi.fedoraproject.org/updates/FEDORA-2023-9f0b552efb)
* [cockpit-ostree Fedora 36](https://bodhi.fedoraproject.org/updates/FEDORA-2023-416be6f1bd)
```

We usually also list the rest it seems.

Comment (garrett) [reply _posts/2023-02-08-cockpit-285.md:43]: The script usually pulls it in? Not sure why it didn't?

Comment (garrett) [reply _posts/2023-02-08-cockpit-285.md:43]: Sure; it works

**Review by jelly — APPROVED** [review]

**Pull request merged**